src/herder/HerderImpl.cpp

```diff
@@ -1132,7 +1158,7 @@ void
 HerderImpl::forceSCPStateIntoSyncWithLastClosedLedger()
 {
     auto const& header = mLedgerManager.getLastClosedLedgerHeader().header;
-    mHerderSCPDriver.restoreSCPState(header.ledgerSeq, header.scpValue);
+    setTrackingSCPState(header.ledgerSeq, header.scpValue, false);
```

Comment:
this should be `true`?
This function is used in "manual close" mode and I don't see a reason to distinguish it from the normal state "we're in sync, everything is fine"